packages/common/src/core/types/IItemEnrichments.ts

```diff
@@ -30,4 +31,9 @@ export interface IItemEnrichments {
    * see https://github.com/ArcGIS/hub-indexer/blob/master/docs/errors.md#response-formatting-for-errors
    */
   errors?: IEnrichmentErrorInfo[];
+
+  /**
+   * Additional resources specified in the formal item metadata
+   */
+  additionalResources?: IHubAdditionalResource[];
```

Comment:
`IItemEnrichments` already has a `metadata` member, should we really add an `additionalResources` member too? I feel like it would through off types for `fetchContent`, but adding it here was the most expedient thing during development.

Reply:
Since Aaron created `IHubEditableContentEnrichments`, this was no longer a problem.